docs/AdvancedFeatures.md

```diff
@@ -150,4 +150,15 @@ An exception to this might be dynamic groups created and maintained by authors o
       allowExternalGroupNamesRegEx: external.* # the AC Tool groups can inherit from other external.* groups
 ```
 
+## Automatically purge obsolete groups
+The root element `obsolete_authorizables` can be used to automatically purge authorizables, that are not in use anymore:
```

Comment:
The doc is talking about authorizables and groups. So is it also possible to delete users? If yes then this should be more clear.

Additionally, an example config should be added to the example content package.